server/controllers/recipe-reaction-controller.js

```diff
@@ -0,0 +1,73 @@
+import db from '../models/index';
+import { successResponse, errorResponse } from '../utils/helpers';
+
+const { Recipe, RecipeRating } = db;
+
+/**
+ *
+ *
+ * @class RecipeReactionController
+ */
+class RecipeReactionController {
+  /**
```

Comment:
There's only one method here and that's for creating/updating ratings - how do I know what the overall rating for a recipe is?